src/JSON.ts

```diff
@@ -34,20 +35,28 @@ export enum ValueKind {
  * The different types of tokens that can be parsed from a JSON string.
  */
 export enum TokenType {
-    LeftCurlyBracket,
-    RightCurlyBracket,
-    LeftSquareBracket,
-    RightSquareBracket,
-    Comma,
-    Colon,
-    Whitespace,
-    QuotedString,
-    Number,
-    Boolean,
-    Literal,
-    Null,
-    Comment,
-    Unrecognized
+    LeftCurlyBracket = 0,
```

Comment:
>= 0, [](start = 21, length = 4)

Curious - why explicit values?

Reply:
To more easily figure out what a numeric value at runtime maps to